src/sirhelpers.c

```diff
@@ -355,14 +355,14 @@ bool _sir_getchar(char* input) {
 # endif
      return true;
 #else /* !__WIN__ */
-    struct termios cur = {0}, new = {0};
+    struct termios cur = {0}, xnew = {0};
```

Comment:
I presume this is because new is a keyword. Let's use tnew

Reply:
Yeah.  It's an error in C++ but it's also a warning in MISRA-C and some other pedantic style enforcements.

Those style guides are interesting though, they don't like variables that are bare adjectives (new, old, good, bad); they want them to be self-describing like old_something, good_thing, etc.

But yeah, I'll make this tnew.